custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomNewNavigationActivity.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.custom.main
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.ActionMode
+import org.kiwix.kiwixmobile.core.base.BaseFragmentActivityExtensions
+import org.kiwix.kiwixmobile.core.di.components.CoreComponent
+import org.kiwix.kiwixmobile.core.main.CoreNewNavigationActivity
+import org.kiwix.kiwixmobile.custom.R
+import org.kiwix.kiwixmobile.custom.customActivityComponent
+
+class CustomNewNavigationActivity : CoreNewNavigationActivity() {
```

Comment:
I don't think we really need CustomNewNavigationActivity because the new navigation will never be available on custom apps.

There should only be NewNavigationActivity(in `app`), both share the same menu xml so Core inflates that. In Custom hide new navigation so it is unclickable see:
```
  override fun onCreateOptionsMenu(menu: Menu?): Boolean {
    val onCreateOptionsMenu = super.onCreateOptionsMenu(menu)
    menu?.findItem(R.id.menu_help)?.isVisible = false
    menu?.findItem(R.id.menu_openfile)?.isVisible = false
    menu?.findItem(R.id.menu_host_books)?.isVisible = false
    return onCreateOptionsMenu
  }
```

for examples where we are hiding items already.

And then in MainMenu set the visibilty of the `new_nav` item based on `BuildConfig.Debug`